resources/user-agents/browsers/chrome/chrome-android-28-to-42.json

```diff
@@ -897,6 +897,13 @@
           "platforms": [
             "Android_6_0"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (*CrKey arm*) applewebkit*(*khtml*like*gecko*) Chrome/#MAJORVER#.*",
```

Comment:
Why was this added in the Android file rather than in the other files with the other patterns?